Makefile

```diff
@@ -45,6 +50,44 @@ validate-infra-secrets: read-keyvault-config install-fetch-config set-azure-acco
 	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -d quiet \
 		&& echo Data in ${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} looks valid
 
+.PHONY: set-space-developer
+set-space-developer: read-deployment-config ## `make dev set-space-developer USER_ID=first.last@digital.education.gov.uk`
+	$(if $(USER_ID), , $(error Missing environment variable "USER_ID", USER_ID required for this command to run))
+	cf set-space-role $(USER_ID) dfe $(SPACE) SpaceDeveloper
+
+.PHONY: unset-space-developer
+unset-space-developer: read-deployment-config ## `make dev unset-space-developer USER_ID=first.last@digital.education.gov.uk`
```

Comment:
Would be cleaner to put comments on their own line rather than in the middle of the commands